engine/runtime-compiler/src/main/scala/org/enso/compiler/phase/ImportResolver.scala

```diff
@@ -61,7 +61,7 @@ final class ImportResolver(compiler: Compiler) extends ImportResolverForIR {
                 u.invalidateCache()
               }
             )
-            compiler.ensureParsed(current, false)
+            compiler.ensureParsed(current, false, false)
```

Comment:
Wouldn't it make sense to have an overloaded method instead? It seems like in a lot of places we don't want to generate docs and this PR propagates that a lot.

Reply:
I don't understand what you mean by "overloaded method". 

> propagates

It sets the value into `ModuleContext` as soon as the context is created.